scripts/config/api-html-artifacts.json

```diff
@@ -30,6 +30,9 @@
     "0.24": "https://ibm.box.com/shared/static/ygdswx8s7fj970kuulkzlh8r2mjuzn3k.zip",
     "0.19": "https://ibm.box.com/shared/static/wjoea4x5tnxd0l4lgo2v3kxnx6btxvvl.zip"
   },
+  "qiskit-c": {
+    "2.0": "https://ibm.box.com/shared/static/0293fd3lwjcee97vr458wiyhebh7row9.zip"
```

Comment:
We'll need to figure out long-term if Qiskit/qiskit will give us two distinct artifacts: Python & C, or we'll have one artifact with both languages. This is sensible to start, though.